_posts/en/newsletters/2021-03-24-newsletter.md

```diff
@@ -0,0 +1,310 @@
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20861][] implements support for [BIP350][] (Bech32m format for v1+
+  witness addresses). Bech32m supersedes bech32 ([BIP173][]) as the address format for
+  native segwit outputs of version 1-16. Native segwit version 0 outputs
+  (P2WPKH and P2WSH) will continue to use bech32. This PR would enable
+  Bitcoin Core users to send payments to Pay to Taproot (P2TR) addresses once taproot outputs ([BIP341][]) were defined on the network.
+  The change should not affect any mainnet systems, but may cause address
+  incompatibility issues in testing environments such as signet where taproot
+  is already active using bech32-encoded addresses as previously proposed.
+  Bech32m support will also be backported to Bitcoin Core 0.19,
+  0.20, and 0.21.
+
+- [Bitcoin Core #21141][] updates the `-walletnotify` configuration
+  setting that calls a user-specified command each time a transaction is
+  seen that affects a loaded wallet.  Two new placeholders are added to
+  the arguments that can be passed to the command, `%b` for the hash of
+  a block containing the transaction and `%h` for the height of the
+  block.  Both are set to defined values for unconfirmed
+  transactions.
+
+- [C-Lightning #4428][] deprecates the `fundchannel_complete` RPC's
+  acceptance of txids, requesting instead that a [PSBT][topic psbt] be
+  passed.  The PSBT can be checked to ensure it contains the funding
+  output, eliminating a [problem][c-lightning #4416] where a user who
+  passes incorrect data can lose the ability to recover their funds.
+
+- [C-Lightning #4421][] implements the funding transaction recovery procedure
+  covered in [last week's newsletter][news140 recovery]. Users who mistakenly
+  funded channels with a first-party malleated transaction (e.g. using RBF) but
+  haven't used the channel yet can now supply their transaction output to the
+  `lightning-close` command to negotiate recovery with a peer supporting the
+  `shutdown_wrong_funding` feature.
+
+- [LND #5068][] makes available a number of new configuration options
+  for limiting how much network gossip information LND processes.  This
+  can help on systems with limited resources.
+
+- [Libsecp256k1 #831][] implements an algorithm that can speed up
+  signature verification by 15%. It can also reduce by 25% the amount of
+  time it takes to generate signatures while still using a constant-time
+  algorithm that maximizes side-channel resistance.  It also reduces
+  Libsecp256k1's dependencies on other libraries.  See [Newsletter
```

Comment:
Style nit: "It can also…" followed by "It also…". Maybe start the second sentence with "The PR additionally reduces…"?